### ISSUE TITLE
babylon should exit with non-zero exit code if error occurs

### ISSUE THREAD

**Issue by dpastoor** [metrumresearchgroup/babylon#143]
if failure occurs babylon should exit with non-zero exit code

```
$stdout
[1] "time=\"2020-03-05T12:20:47-05:00\" level=info msg=\"Successfully loaded default configuration from /Users/devinp/clients/metrum/tech-solutions/rbabylon-example-project/model/pk/babylon.yaml\""                                                                                                                     
[2] "time=\"2020-03-05T12:20:47-05:00\" level=info msg=\"Beginning Local Path\""                                                                                                                                                                                                                                          
[3] "time=\"2020-03-05T12:20:47-05:00\" level=info msg=\"A total of 1 models have completed the initial preparation phase\""                                                                                                                                                                                              
[4] "time=\"2020-03-05T12:20:47-05:00\" level=error msg=\"1 errors were experienced during the run\""                                                                                                                                                                                                                     
[5] "time=\"2020-03-05T12:20:47-05:00\" level=error msg=\"Errors were experienced while running model 102. Details are The target directory, /Users/devinp/clients/metrum/tech-solutions/rbabylon-example-project/model/pk/102 already exist, but we are configured not to overwrite. Invalid configuration / run state\""
[6] "time=\"2020-03-05T12:20:47-05:00\" level=info msg=\"\\r1 models completed in 5.67287ms\""                                                                                                                                                                                                                            
[7] "" 
```

Other tooling, such as rbabylon, can use the error code to inspec whether the action was successful from a user perspective

**Comment by shairozan** [comment]
Available in https://github.com/metrumresearchgroup/babylon/releases/tag/v2.1.0-rc2

**Comment by shairozan** [comment]
Released in version v2.1.0
